task_DEV_6/task_DEV_6/Stack.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+
+namespace task_DEV_6
+{
+    class Stack
+    {        
+        public int[] stack;
+        public int count;
+
+        public Stack()
+        {
+            stack = new int[10];
+        }
+        public Stack(int number)
+        {
+            stack = new int[number];
+        }
+
+        public int Capacity()
+        {
+            return stack.Length;
+        }
+
+        public int Count()
+        {
+            return count;
+        }
+
+        public bool IsEmpty()
+        {
+            if (count == 0)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        public void NegativeResize()
```

Comment:
Do we really need to split one method `Resize` to `Positive` and `Negative`?

Reply:
Done.